admin/index_m.html

```diff
@@ -101,7 +101,7 @@
                     return;
                 }
 
-                $('#dialog-bridges .modal-content').html(_('Please wait, searching for bridge.<br>' +
+                $('#dialog-bridges .modal-content').html(_('Please wait, searching for bridge.' + '<br>' +

```

Comment:
```suggestion
$('#dialog-bridges .modal-content').html(_('Please wait, searching for bridge.' ) + '<br>' +
                    '<div class="progress">' +
                    '   <div class="indeterminate"></div>' +
                    '</div>');
```

should be correct. Or is it tested?

Reply:
You are right, it was just a quick change in the browser (untested). Thanks for the review 👍

Reply:
Please check the closing brackets, too. This was the issue.